README.md

```diff
@@ -218,6 +218,49 @@ $client = KeenIOClient::factory([
 $results = $client->getSavedQuery(['query_name' => 'total-API-requests']);
 ```
 
+#### Using Cached queries
+
+By [Caching a Query](https://keen.io/docs/api/?php#creating-a-cached-query), you are updating the query you have saved earlier by adding a `refresh_rate` property.
```

Comment:
You can create a cached query from the start too. You don't have to update an existing saved query to do it. There might be a way to make this more clear here.

Reply:
Caching a query isn't updating it. Caching a query is setting the refresh_rate through updating it.